src/fspec-iso.c

```diff
@@ -2,6 +2,7 @@
 #include <stdlib.h>
 #include <libgen.h>
 #include <err.h>
+#include <string.h>
```

Comment:
I currently have no convention at all lol. 

I thought include order sometimes affects semantics? Anyway your system is what I follow for when I'm using Go more or less.

Reply:
In the end I just sorted all <> headers in one block and "" headers in another block. We can revisit, this is the sort of thing I need a tool for otherwise I will just keep making mistakes.

Reply:
I believe the ordering of libc headers doesn't matter. Kernel headers should always come after libc headers, but we aren't using any of those.

Reply:
Sounds good.